handler/class_test.go

```diff
@@ -396,3 +395,167 @@ func TestDeleteClass(t *testing.T) {
 		}
 	})
 }
+
+func TestJoinClass(t *testing.T) {
+	t.Run("validJoin", func(t *testing.T) {
+		d := db.OpenMock()
+		require.NoError(t, d.StoreClass(context.Background(), db.Class{
+			CID: "test",
+		}))
+		require.NoError(t, d.StoreUser(context.Background(), db.User{
+			UID: "test",
+		}))
+
+		req := httptest.NewRequest(http.MethodPost, "/", strings.NewReader(`{"uid": "test", "cid": "test"}`))
+		rec := httptest.NewRecorder()
+		assert.NotNil(t, req, rec)
+		c := echo.New().NewContext(req, rec)
+
+		if assert.NoError(t, handler.JoinClass(&db.DBContext{
+			Context: c,
+			TLADB:   d,
+		})) {
+			require.Equal(t, http.StatusOK, rec.Code)
```

Comment:
Small nit: check for something being returned (the class data)

Reply:
I added tests verifying the class data is returned